src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,103 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Class containing the main Duke application.
+ * Initialises the application.
+ */
+public class Duke {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        Storage st = new Storage("./data/dukedata.txt");
+        TaskList tl = new TaskList(st.setFile());
+        Ui ui = new Ui();
+
+        ui.printIntro();
+
+        while (true) {
```

Comment:
I think it will be better if you move this part to parser, like process the commands in paser. 😸